datatypes/src/raster/operations/convert_data_type.rs

```diff
@@ -55,6 +55,7 @@ where
             global_geo_transform: self.global_geo_transform,
             properties: self.properties,
             tile_position: self.tile_position,
+            cache_hint: self.cache_hint, // TODO: update created date?
```

Comment:
But we do not really do updates, but rather create new tiles. We could maybe just do CacheHint::from(self.cache_hint.expiration())` here, because that uses the current system date time.

Reply:
Sure, we can do that.